src/main/java/fr/inria/coming/codefeatures/P4JFeatureAnalyzer.java

```diff
@@ -68,11 +68,18 @@ public AnalysisResult analyze(IRevision revision, RevisionResult previousResults
 		Option option = new Option();
 		option.featureOption = FeatureOption.ORIGINAL;
 		//We set the first parameter of CodeDiffer as False to not allow the code generation at buggy location
-		CodeDiffer codeDiffer = new CodeDiffer(false, option);
+		//The second false indicates cross features
+		Boolean cross = false;
```

Comment:
Even for the moment we will only use the NonCross, it's recommended to avoid dead code (as the line 79).
One option to control the value of `cross` variable (and thus to decide if execute cross or not) could be to get the value from the configuration file:
`ComingProperties.getPropertyBoolean("cross")`

The default value would be in the file `config-coming.properties` as:
`cross=false`

Note that this is the `fastest` way to pass that value: a more elegant solution would be to pass `cross` as a boolean argument, but that would demand to change several methods and calls

Reply:
Hi @martinezmatias, 
That's an elegant way! I will work in this way in the next commits.